operatorcourier/verify.py

```diff
@@ -0,0 +1,143 @@
+import os
+import logging
+from typing import List, Dict
+import semver
+from operatorcourier.build import BuildCmd
+from operatorcourier.validate import ValidateCmd
+from operatorcourier.errors import OpCourierBadBundle
+from operatorcourier.format import format_bundle
+from operatorcourier import identify
+
+logger = logging.getLogger(__name__)
+FLAT_KEY = '__flat__'
+
+
+class Verification:
+    def __init__(self, source_dir, yamls, ui_validate_io, repository):
+        manifests = self.get_manifests_info(source_dir, yamls)
+        self.nested = len(manifests) > 1
+        self.bundle_dict = None
+        self.validation_dict = \
+            self.get_validation_dict_from_manifests(manifests, ui_validate_io, repository)
+        if not self.nested:
+            self.bundle = format_bundle(self.bundle_dict)
+
+    def get_manifest_files_content(self, file_paths: List) -> List:
```

Comment:
@kevinrizza Since Python is dynamically typed, I just think that when it's not obvious what some params / return values' type is, it may help to explicitly add type hints.

Reply:
@jeremylinlin sure, but we should follow a particular style across the entire codebase. If we want to introduce a change like this, we should do it across every single function in the entire package -- and that is outside the scope of this PR.